scripts/bundles/compiler.ts

```diff
@@ -228,3 +227,12 @@ async function minifyStencilCompiler(code: string, opts: BuildOptions) {
 
   return code;
 }
+
+/**
+ * Helper function that reads in the `lib.*.d.ts` files in the TypeScript lib/ directory on disk.
+ * @param opts the Stencil build options, which includes the location of the TypeScript lib/
+ * @returns all file names that match the `lib.*.d.ts` format
+ */
+async function getTypeScriptDefaultLibNames(opts: BuildOptions): Promise<string[]> {
```

Comment:
Original is from https://github.com/ionic-team/stencil/pull/4066/files#diff-66d383a94d79e7a771f77d62e1f6cb976c5e096f8cd8f70f74fb162bd3142af3L85